exporters/FusionExporter/Source/AddIn/EUI-Handlers.cpp

```diff
@@ -135,3 +135,30 @@ bool EUI::clearHandler<CloseExporterFormEventHandler>(Ptr<Palette> palette)
 	
 	return closeEvent->remove(closeExporterFormEventHandler);
 }
+
+template<>
+bool EUI::addHandler<CloseGuideFormEventHandler>(Ptr<Palette> palette)
```

Comment:
Instead of creating another event handler for each close event, let's try to use a similar method for sharing handler classes as we do for button press events. I recently did this in b0765d1381e0b1a4de4d47e0f994c3458a7da0ab, so it might be easiest to merge `1029-fusion-export-ui` into this branch and add another condition here: https://github.com/Autodesk/synthesis/blob/b0765d1381e0b1a4de4d47e0f994c3458a7da0ab/exporters/FusionExporter/Source/AddIn/CustomHandlers.cpp#L95-L101

Reply:
Actually, since I was just editing this event structure anyways, I could also make this edit on this branch if you want.

Reply:
Go for it, thanks!